config/locales/en.yml

```diff
@@ -570,11 +570,16 @@ en:
       plugin_php: PHP Plugin
       plugin_rest: REST requests
       plugin_csharp: .NET Plugin
+      service_mesh_istio: Istio Service Mesh
 
     self_managed: APIcast self-managed<span>Manage your own NGINX based API gateway for ultimate flexibility and customization.</span>
     hosted: APIcast<span>a 3scale managed NGINX based API gateway for quick and reliable integrations</span>
     apicast: APIcast<span>Nginx reverse proxy server.</span>
 
+    service_mesh_istio: |
```

Comment:
I think this should be `service_mesh_istio` as it contains a `span` tag.
If not suffixed then it will be escaped by Rails

Reply:
@hallelujah and I checked and it works. It is called from `app/views/api/integrations/apicast/shared/_deployment_options.html.slim:40` with `dd.u-dl-definition == t(@service.proxy.deployment_option, scope: :deployment_options)`
![image](https://user-images.githubusercontent.com/11318903/48422085-2e07b800-e75e-11e8-9911-a32e93d08858.png)